homeassistant/components/linkplay/__init__.py

```diff
@@ -0,0 +1,44 @@
+"""Support for LinkPlay devices."""
+
+from linkplay.bridge import LinkPlayBridge
+from linkplay.discovery import linkplay_factory_bridge
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import PLATFORMS
+
+
+class LinkPlayData:
```

Comment:
We normally use a dataclass for this.

https://docs.python.org/3/library/dataclasses.html

Reply:
Addressed in #124307